java/client/src/main/java/glide/api/commands/GenericClusterCommands.java

```diff
@@ -251,6 +250,65 @@ public interface GenericClusterCommands {
      */
     CompletableFuture<Object[]> scan(ClusterScanCursor cursor);
 
+    /**
+     * Incrementally iterates over the keys in the Cluster.
+     *
+     * <p>This command is similar to the <code>SCAN</code> command, but it is designed to work in a
+     * Cluster environment. The main difference is that this command uses a {@link ClusterScanCursor}
+     * object to manage iterations. For more information about the Cluster Scan implementation, see <a
+     * href="https://github.com/aws/glide-for-redis/wiki/General-Concepts#cluster-scan">Cluster
```

Comment:
The doc linked here has a TODO at the bottom still. Is that something we should address before release?

Reply:
I don't know.  That's Avi's document?  Might we worthwhile taking a look next week.